MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+
+By default label position is left.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+                e.DataFormItem.DefaultLayoutSettings.LabelWidth = 100;
+                e.DataFormItem.DefaultLayoutSettings.EditorWidth = 200;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Background color
+
+You can change the background color for the label and editor by using `GenereateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Brush.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+#### Setting order for editor
+
+You can change the row order of an item using `RowOrder` property of `DataFormViewItem`.
+
+You can change the order of item's within a row using `ItemsOrderInRow` property, which is applicable only when `ColumnCount` is set to more than one. You can place an item within a row from 0 to `ColumnCount` - 1 position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ColumnCount = 3;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 1;
+                e.DataFormItem.ItemsOrderInRow = 2;  
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Label icon
+
+You can set label icon instead of using label text. 
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "PhoneNumer")
+            {
+                e.DataFormItem.LeadingLabelIcon = new FontImageSource() 
+                { 
+                    Glyph = "K", 
+                    Color = Colors.Black, 
+                    FontFamily = "DataFormIcons", 
+                    Size=18 
+                };
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing Text style 
```

Comment:
## Change the text style

You can change the text style for label text, editor text , valid message text and error message text by using GenereateDataFormItem event when AutoGenarte is true.

For AutoGenerate false you can use property directly by using the DataFormItem - add refrence link for explict create data editor

Reply:
content and  title added